### PR TITLE
Update docs examples with latest dependency versions

### PR DESCRIPTION
**Pull request by nizarmah**
Updates the docs examples in the README with the latest dependency versions, as suggested in #18.

### CODE REVIEW

**Comment by nizarmah** [comment]
Hello @Spreeuw 😄 I created a pull request, as you requested. Would you like to review this change? 

I'll give you a bit of time to answer/review, until Wednesday August 18; then, I'll merge the changes.

**Comment by nizarmah** [comment]
Please note that there are some changes to the README in #19. The changes there match the current pull request. 👍🏼

File: README.md
Comment (Spreeuw) [inline README.md:28]: Unless you have a specific reason to restrict it to `.0` I would suggest to just use `@v2`
(same for the other occurrences)

Comment (nizarmah) [reply README.md:28]: @Spreeuw I do, I'm planning to release `v2.1` on Wednesday once #19 is merged.

Comment (Spreeuw) [reply README.md:28]: I'm not sure that answers my question 😉 
If you follow SemVer, `v2.1` would not have any breaking changes compared to `v2.0`, so then it would make sense to use `@v2` in your docs. If you **_do not_** follow SemVer and `v2.1` is a breaking change... Then you should indeed use `@v2.0` in your docs.

Comment (nizarmah) [reply README.md]: Oh! @Spreeuw thanks for the clarification 👀 I've updated it to `@v2` since `v2.1` isn't a breaking change.

I'll read more about semantic versioning; I wasn't aware of it so thanks for pointing me in the right direction 😃

**Review by Spreeuw — APPROVED** [review]
🚀

**Comment by Spreeuw** [comment]
@nizarmah you mentioned here that you were planning to release v2.1 last Wednesday, is anything holding up the release?

**Comment by nizarmah** [comment]
@Spreeuw Yeah! Sorry about that. I was having power issues in my country, which obstructed some of my plans during those times I'm available for my open source work. 

I'm doing that in the next hour. It's a high priority on my list 👍🏼

**Comment by Spreeuw** [comment]
Great, thank you so much for the release! One nitpick:
Your release is named 'Add option for minified files to overwrite existing files' but that looks a bit strange in the 'releases' section:
auto-minify-release-name

It's more common to include the version number in the release name/title, like [github themselves](https://github.com/github/fetch) do too:
github-fetch-release-name

That also addresses another nitpick:
Your release changelog doesn't show all changes, currently it only shows:

- Add option for minified files to overwrite existing ones (#19) 

but it also includes:

-  Fixes issue with file names ending in "min" from being skipped (#17) 
-  Update docs examples with latest dependency versions (#20) 

(you may or may not want to include the docs change, that's up to you of course)

**Comment by nizarmah** [comment]
@Spreeuw 

> It's more common to include the version number in the release name/title, like github themselves do too

That's good to know! Thank you! 

I never knew what to include in the Release Title. And lately due to my power/connection issues in my country, I haven't had the chance to educate myself on those aspects 😔 

> Your release changelog doesn't show all changes

Yeah, I tend to forget that other people are using my tools. I love open source, but I'm not used to having an audience 😅 

I often leave my changelog empty, sadly. I'll do my best to take things more seriously now that my tools are being used by others. 

---

Also, I really appreciate your help with this. There's a lot of stuff I don't know and I haven't been reading about because I haven't had much of an audience. Your comments are really helping push me into the right direction, during this time. 

At the moment, unfortunately, I've been spread thin across real world issues, but once those are in the clear, I'll have more time to deliver better quality in my repositories 😃

**Comment by Spreeuw** [comment]
No worries at all, I really appreciate your work on this, just trying to contribute!

**Comment by nizarmah** [comment]
I'm glad you are @Spreeuw 🤗 Hope you have a wonderful day btw! 🚀

**Pull request merged**
